homeassistant/components/tradfri/light.py

```diff
@@ -14,20 +12,19 @@
     SUPPORT_COLOR,
     SUPPORT_COLOR_TEMP,
 )
-from homeassistant.core import callback
 from .base_class import TradfriBaseDevice
 from .const import (
     ATTR_DIMMER,
     ATTR_HUE,
     ATTR_SAT,
     ATTR_TRANSITION_TIME,
     SUPPORTED_LIGHT_FEATURES,
-    SUPPORTED_GROUP_FEATURES,
     CONF_GATEWAY_ID,
     CONF_IMPORT_GROUPS,
     KEY_GATEWAY,
     KEY_API,
 )
+from .light_group import TradfriLightGroup
```

Comment:
Why do you want to move the entity to another module? Both entity types are added to home assistant in this module.

It doesn't seem consistent to have one of the types in a different module.

Reply:
It’s a matter of personal preference of file handling I guess. I’m happy to close this PR if it’s against best practice.

Can the adding of modules be centralized or do they have to be in each file?

Reply:
Do you mean adding of *entities*?

Each entity platform will add its own type of platform entity.

Reply:
Yes, sorry, I meant putting `async def async_setup_entry` from each device-type file in the Tradfri lib into __init__.py.